src/marshaller/HipieDDL.js

```diff
@@ -1008,12 +1010,13 @@
                 });
                 break;
             case "GRAPH":
+            case "GRAPHC":
```

Comment:
Not needed (GRAPH is a HIPIE keyword, user will set the _chartType("graph_GraohC" for this graph)